File: go/tax_breakdown_item.go
// this file is @generated
package meteroid

type TaxBreakdownItem struct {
	// Free-text legal exemption mention (EU exempt/reverse-charge invoices).
	ExemptionReason *string `json:"exemption_reason,omitempty"`

	ExemptionType *TaxExemptionType `json:"exemption_type,omitempty"`

	Name string `json:"name"`

	TaxAmount int64 `json:"tax_amount"`

	TaxRate string `json:"tax_rate"`

	// Accounting/reporting code of the tax rate for this line, for exports.
	TaxReference *string `json:"tax_reference,omitempty"`

	TaxableAmount int64 `json:"taxable_amount"`
}

File: python/meteroid/models/tax_breakdown_item.py
# this file is @generated
import dataclasses
import typing as t
from decimal import Decimal

from ..serialization import BaseModel
from .tax_exemption_type import TaxExemptionType


@dataclasses.dataclass
class TaxBreakdownItem(BaseModel):
    name: str

    tax_amount: int

    tax_rate: Decimal

    taxable_amount: int

    exemption_reason: t.Optional[str] = None
    """Free-text legal exemption mention (EU exempt/reverse-charge invoices)."""

    exemption_type: t.Optional[TaxExemptionType] = None

    tax_reference: t.Optional[str] = None
    """Accounting/reporting code of the tax rate for this line, for exports."""
